### DISCUSSION TITLE
Possible to import existing cluster?

### DISCUSSION THREAD

**Discussion by mubarak-j** [kubernetes-sigs/cluster-api#7716] [Q&A] [answered]
I'm new CAPI and I'm curious if there's a way to manage an existing cluster that was created by a different tool (e.g eksctl)?

**Comment by killianmuldoon** [comment, marked as answer]
There's no CAPI-supported way to do this currently, but there is some prior art in the community - e.g. https://www.youtube.com/watch?v=KzYV-fJ_wH0

There has also been a bit of discussion around this in the community meeting,  and there's an open issue with some additional insight over at the CAPZ repo - https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/1173

You can see the discussion in the August 10th community meeting here: https://youtu.be/x22eYJheDxI?t=993
